genesyscloud/telephony_providers_edges_site_outbound_route/resource_genesyscloud_telephony_providers_edges_site_outbound_route_test.go

```diff
@@ -0,0 +1,226 @@
+package telephony_providers_edges_site_outbound_route
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"os"
+	"strconv"
+	"strings"
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/telephony"
+	telephonyProvidersEdgesSite "terraform-provider-genesyscloud/genesyscloud/telephony_providers_edges_site"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	featureToggles "terraform-provider-genesyscloud/genesyscloud/util/feature_toggles"
+	"testing"
+)
+
+func TestAccResourceSiteOutboundRoutes(t *testing.T) {
+	t.Parallel()
```

Comment:
If you're going to remove the t.Skip() from the old tests, then you'd want to remove this t.Parallel line and add a defer func to unset the env variable